executable/installer/iexpress/packaged-as-an-iexpress-self-extracting-archive.yml

```diff
@@ -0,0 +1,18 @@
+rule:
+  meta:
+    name: packaged as an IExpress self-extracting archive
+    namespace: executable/installer/iexpress
+    author: '@recvfrom'
+    scope: file
+    references:
+      - https://en.wikipedia.org/wiki/IExpress
+    examples:
+      - ac742739cae0d411dfcb78ae99a7baee:0xA4C0 # wextract_cleanup%d
+      - ac742739cae0d411dfcb78ae99a7baee:0xA488 # Software\Microsoft\Windows\CurrentVersion\RunOnce
+      - ac742739cae0d411dfcb78ae99a7baee:0x34BA2 # '  <description>IExpress extraction tool</description>'
```

Comment:
i'm not sure that we have other file scope examples as file offsets, but this makes sense to me